gamestonk_terminal/etf/README.md

```diff
@@ -120,53 +122,48 @@ DOGE not found
 ```
 ## screener <a name="screener"></a>
 ````python
-usage: screener [--config] [-p MIN_PRICE] [-P MAX_PRICE] [-a MIN_ASSETS] [-A MAX_ASSETS] [-n MIN_NAV] [-N MAX_NAV] [-e MIN_EXP] [-E MAX_EXP] [-r MIN_PE] [-R MAX_PE] [-d MIN_DIV]
-              [-D MAX_DIV] [-b MIN_BETA] [-B MAX_BETA] [--num] [--export {csv,json,xlsx}] [-h]
+usage: screener  [-n --num] [--export {csv,json,xlsx}] [-h]
```

Comment:
Can we refactor this command to have a `presets` folders? Similar to how `insider`, `screener` and `options` have.

This allows us to build a database of user presets which makes it more useful than having to edit 1 single file.